src/core/messages.js

```diff
@@ -1,23 +1,24 @@
 const messages = [
-  "Nice job 👍",
-  "Looks good 👍",
-  "Clean fix! ✨",
-  "Well done 👍",
-  "Thank you! Really nice PR ❤️",
-  "Great job 👍",
-  "Looks amazing 🔥",
-  "Thank you! I'm so happy that we finally did this 😍",
-  "Looks nice! 😍",
-  "Nice refactor! 💪",
+  "Nice job",
+  "Looks good",
+  "Clean fix!",
+  "Well done",
+  "Thank you! Really nice PR",
+  "Great job",
+  "Looks amazing",
+  "Thank you! I'm so happy that we finally did this",
+  "Looks nice!",
+  "Nice refactor!",
   "Amazing Job 🥇 It was a complex task you've pulled off!",
-  "Great work on this! 🚀 💯",
-  "Looking good so far 🎉",
+  "Great work on this!",
+  "Looking good so far",
   "I think this is a good change",
-  "This is looking ✨",
-  "Superb work! You completed a challenging task 🔥",
-  "Excellent work 👍🏼",
-  "This is an excellent improvement 💯",
-  "Nice and tidy, great job! 💪",
+  "This is looking",
+  "Superb work! You completed a challenging task",
+  "Excellent work",
+  "This is an excellent improvement",
+  "Nice and tidy, great job!",
 ];
+const emojis = ["👍", "✨", "❤️", "🔥", "😍", "💪", "🥇", "🚀", "🎉", " "];
```

Comment:
Can we remove `" "`? I know the point is to have empty value at the begginning, but let's add this for all messages. And can you do this randomly as it's done in message generator? At the moment we take them by index.